lib/tictactoe/cli/player_menu.ex

```diff
@@ -0,0 +1,62 @@
+defmodule TicTacToe.CLI.PlayerMenu do
```

Comment:
I'm wondering if we can choose a different name for this module? Yes, it's displaying the menu, but it looks like the main responsibility for this module is to set up the game depending on the player selection.

Reply:
Ah ok, I see what you mean. I'll change it to GameConfig so that it can expand in the future if there are other options that the player needs to select